services/visual_recognition/v3.js

```diff
@@ -171,8 +163,7 @@ module.exports = function (RED) {
 
         if (err != null && body==null)
         {
-          node.status({fill:'red', shape:'ring', 
-                       text:'call to watson visual recognition v3 service failed'}); 
+          node.status({fill:'red', shape:'ring', text:'call to watson visual recognition v3 service failed'}); 
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Line is too long.](https://www.codacy.com/app/BetaWorks-NodeRED-Watson/node-red-node-watson/file/2712969112/issues/source?bid=3330694&fileBranchId=3344618#l166)